doc/sphinx/Pacemaker_Explained/cluster-options.rst

```diff
@@ -62,6 +62,39 @@ Normally, you will use command-line tools that abstract the XML, so the
 distinction will be unimportant; both properties and options are cluster
 settings you can tweak.
 
+.. note::
+
+   If there are multiple ``nvpair`` options with the same name with the same
+   parent element, the first one is applied and the rest are ignored. Similarly,
+   if there are multiple options with the same name in a group of parent
+   elements with the same score within a given grandparent element, the first
+   one is applied and the rest are ignored.
```

Comment:
Oddly enough we don't use "parent" anywhere else in Pacemaker Explained to refer to XML. Maybe:

If one enclosing element, or multiple such elements with equal scores, have more than one ``nvpair`` option with the same ``name``, the first one listed is used and the rest ignored.

Reply:
On the other hand, I feel like anyone fiddling with XML ought to know what "parent" means at the very least... but always open to feedback on docs, not my forte

Reply:
In a case like this, either terminology is fine, but sticking with one consistently helps people tie together different strands